### ISSUE TITLE
Python 3.4 `JSONDecodeError` import error

### ISSUE THREAD

**Issue by mortezaipo** [mortezaipo/samplecmd#3]
System info: `centos 7`
Python: `3.4`
```
$ samplecmd docker
Traceback (most recent call last):
File "/usr/bin/samplecmd", line 9, in <module>
load_entry_point('samplecmd==1.1.1', 'console_scripts', 'samplecmd')()
File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 568, in load_entry_point
return get_distribution(dist).load_entry_point(group, name)
File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2720, in load_entry_point
return ep.load()
File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2380, in load
return self.resolve()
File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2386, in resolve
module = __import__(self.module_name, fromlist=['__name__'], level=0)
File "/usr/lib/python3.4/site-packages/samplecmd/samplecmd.py", line 31, in <module>
from json.decoder import JSONDecodeError
ImportError: cannot import name 'JSONDecodeError'
```

**Comment by mortezaipo** [comment]
According to Python3.5 change log, in Python3.4 we should handle `ValueError` instead of `JSONDecodeError`.
https://docs.python.org/3.5/whatsnew/changelog.html#id93
https://bugs.python.org/issue19361

**Comment by mortezaipo** [comment]
Bug fixes in the last commit: [a9a5596](https://github.com/mortezaipo/samplecmd/commit/a9a5596d1283a9001499c32c2ba535292daa9348)
